public_suffix_list.dat

```diff
@@ -15290,6 +15290,10 @@ reserve-online.com
 bookonline.app
 hotelwithflight.com
 
+// WebWaddle Ltd: https://webwaddle.com/
+// Submitted by Merlin Glander <merlin@webwaddle.com>
```

Comment:
If possible, please use a non-personal e-mail so it stays reachable.

Reply:
changed now to hostmaster address